hermes-console-vue/json-server/server.js

```diff
@@ -0,0 +1,27 @@
+const topics = require('./topics.json');
+const subscriptions = require('./subscriptions.json');
+const routes = require('./routes.json');
+
+const jsonServer = require('json-server');
+const server = jsonServer.create();
+
+const middlewares = jsonServer.defaults();
+server.use(middlewares);
+
+const rewriter = jsonServer.rewriter(routes);
+server.use(rewriter);
+
+server.post('/query/topics', (req, res) => {
+  res.jsonp(topics);
+});
+
+server.post('/query/subscriptions', (req, res) => {
+  res.jsonp(subscriptions);
+});
+
+const router = jsonServer.router('json-server/db.json');
+server.use(router);
+
+server.listen(3000, () => {
+  console.log('JSON Server is running');
+});
```

Comment:
As i'm looking with Mateusz at this endpoint, I think it would be best to simply add a new endpoint in hermes-management to return necessary data, instead of returning huge json with all topics, and counting them on frontend. What do you think? Just two of those endpoints that returns topics and subscriptions are returning 12MB (sic!) of data.

Reply:
Yeah I agree, I was also surprised by how its implemented right now